mscore/plugin/api/elements.h

```diff
@@ -347,6 +348,11 @@ class Note : public Element {
 
       QQmlListProperty<Element> dots()     { return wrapContainerProperty<Element>(this, note()->dots()); }
       QQmlListProperty<Element> elements() { return wrapContainerProperty<Element>(this, note()->el());   }
+
+      Q_INVOKABLE Element* accidental() { return wrap<Element>(note()->accidental()); }
```

Comment:
As far as I know `Q_INVOKABLE` functions should have a fully specified namespace in its return type (like `Ms::PluginAPI::Element*`) in order to be processed correctly by moc. At least things did not work correctly for me when I did not specify namespace explicitly. Does this option work for you correctly?

Reply:
Thanks for spotting this: `Q_INVOKABLE` seems simply wrong here. It's a `Q_PROPERTY` helper function and doesn't need to be called from a plugin. Therefore I didn't try it and I don't know if it would have worked. I'll remove it.